ml/src/main/scala/ai/h2o/sparkling/ml/models/H2OTargetEncoderModel.scala

```diff
@@ -38,9 +38,9 @@ class H2OTargetEncoderModel(override val uid: String, targetEncoderModel: H2OMod
   with RestCommunication {
 
   lazy val mojoModel: H2OTargetEncoderMOJOModel = {
-    val mojoData = targetEncoderModel.downloadMojoData()
+    val mojo = targetEncoderModel.downloadMojo()
     val model = new H2OTargetEncoderMOJOModel()
-    copyValues(model).setMojoData(mojoData)
+    copyValues(model).setMojoData(mojo)
```

Comment:
The name of this method should be changed as well to reflect that we are using MOJO, not mojo data